### PR TITLE
fix(cli): handle pnpm build approvals

### PR DESCRIPTION
**Pull request by coryrylan**
## Summary by CodeRabbit

* **Documentation**
  * Updated Codex/MCP setup examples and site docs with richer TOML snippets including server descriptions.

* **Bug Fixes**
  * Better package-manager handling: ignored build-script errors are non-fatal and failure output is normalized.
  * Improved dropdown close behavior and ensured single-select validation runs on close.

* **Style**
  * Disabled pointer interactions for multi-select option elements; applied global select styles.

* **Chores**
  * Loosened peer dependency version ranges for broader compatibility.

* **Tests**
  * Expanded test coverage across config, package-manager, utilities, and edge cases.

### CODE REVIEW

File: projects/core/src/combobox/combobox.ts [hunk 1]
```diff
@@ -371,8 +371,14 @@ export class Combobox extends Control implements ContainerElement {
...
 
-  #onDropdownOpen(e: Event) {
-    (e.target as HTMLElement).hidden = false;
+  #openDropdown() {
+    this.#dropdown!.hidden = false;
+  }
+
+  #closeDropdown() {
+    this.#dropdown!.hidden = true;
+    this._internals.states.delete('dirty');
```
Comment (johnyanarella) [inline projects/core/src/combobox/combobox.ts:380]: I get the sense there are two competing meanings of "dirty" at play here.

1) the traditional "this field was changed vs the initial state" per `Control` where any `input` event sets this state (and form reset clears it).
2) a transient (while open) state unique to the combobox around the search and highlighting

Maybe this needs a distinct state? Otherwise, unconditionally clearing on close makes this behave differently than other controls.

**Review by johnyanarella — APPROVED** [review]

File: projects/core/src/combobox/combobox.ts (hunk 1 above)
Comment (coryrylan) [reply projects/core/src/combobox/combobox.ts:380]: Yeah that makes sense, I was thinking in terms of "active" it shouldn't be treated as "dirty". I'll fix that

Comment (coryrylan) [reply projects/core/src/combobox/combobox.ts:380]: Ok, going to likely follow up on that change since dirty was the prior behavior. It looks like thats used to determine the initial filtering state on open. I need to figure out the exact logic that the base form control has that sets dirty and emulate that here to ensure it doesnt break any prior behavior

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)

**Comment by coryrylan** [comment]
🎉 This issue has been resolved in version 2.0.1 🎉

[Changelog](https://NVIDIA.github.io/elements/docs/changelog/)
